doc/tutorials/using_zephyr_as_uos.rst

```diff
@@ -0,0 +1,88 @@
+.. _using_zephyr_as_uos:
+
+Using Zephyr as User OS
+#######################
+
+The tutorial describes the steps to run zephyr as User OS on ACRN hypervisor. We are using
+Kaby Lake-based NUC model NUC7i5DNH and you can try other platform ACRN supported as well.
+
+Introduction to Zephyr
+**********************
+
+The Zephyr is a scalable real-time operating-system (RTOS) supporting multiple hardware architecture,
+optimized for resource constrained devices, and built with safety and security in mind.
+
+Steps for Using Zephyr as User OS
+*********************************
+
+#. Build Zephyr and grub2 boot loader image
+
+   First of all, please follow the `Development Environment Setup on Linux
+   <https://docs.zephyrproject.org/latest/getting_started/installation_linux.html>`_ to
+   setup Zephyr development environment. Then follow the `Board-Specfic Support
+   <https://docs.zephyrproject.org/latest/boards/index.html>`_ to build the acrn supported zephyr image ``zephyr.elf``.
```

Comment:
There is no such reference to `zephyr.elf` on this page (or even some sort of an ACRN BSP page). Can you add a more accurate pointer to where one can find info about the next steps?

@dbkinder , I'm actually thinking that having an `ACRN` bsp page on Zephyr would be good, what do you think?

Reply:
@gvancuts, there are patches in zephyr side containing one doc about how to build zephyr.elf which can run on top of acrn. I will update the reference with that doc once those patches got merged in zephyr upstreaming tree.

Reply:
Great! Do you have a link to the PR yet?